net-proxy/3proxy/3proxy-0.7.1.2.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 1999-2012 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+EAPI=5
+inherit eutils
+
+DESCRIPTION="3Proxy is really tiny cross-platform proxy servers set"
```

Comment:
There's really no reason to repeat program name in `DESCRIPTION`, i.e. `="A really tiny cross-platform ..."`.